src/frontend/src/pages/ProjectPage/pages/ContractPage/index.tsx

```diff
@@ -86,7 +86,7 @@ const ContractPage: React.FC<ContractPageProps> = ({ match }) => {
                     <div className={styles.details}>
                         <ScopedSwitch>
                             <Route path="/" exact component={ContractDetailsPage} />
-                            <Route path="/personnel" component={ManagePersonellPage} />
+                            <Route path="/managepersonnel" component={ManagePersonellPage} />
```

Comment:
Since the rest of the routes uses dashes to seperate words, we should probably keep doing that